### ISSUE TITLE
Review annotations to GO:0005665 DNA-directed RNA polymerase II, core complex

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#2020]
Some annotations to GO:0005665 DNA-directed RNA polymerase II, core complex seem incorrect, please review: 

Group | Number annotations
-- | --
dictyBase | 1
GeneDB | 9 @uliboehme
MGI | 1
SGD | 1
UniProt | 6

https://docs.google.com/spreadsheets/d/1-Gnjt_8rbci9XwCc2Dd4w7JGVpwTFcpvof93WMjLqHY/edit#gid=0

**Comment by krchristie** [comment]
@pgaudet - You have suggested changing some annotations to "protein complex binding", but there already exists a specific term for "RNA polymerase II core binding" (GO:0000993), which seems likely to be the appropriate term in these cases.

**Comment by pgaudet** [comment]
@krchristie this probably works in some cases; where I put that suggestions, I thought proteines were *members* of the complex, so they should NOT be annotated to it - right ? 

Thanks, Pascale

**Comment by krchristie** [comment]
I thought the annotations you were suggesting were incorrect were the proteins that are NOT part of the core complex, i.e. the 12 subunit complex. I agree that these should NOT be annotated to this term. However, if they are interacting with the core complex, then it DOES seem appropriate to annotate to the term "RNA polymerase II core binding" (GO:0000993). This is what I did for the single MGI annotation to this term you flagged in this spreadsheet.

**Comment by suzialeksander** [comment]
Hi,
Until there's a definite repeal of the "colocalizes_with", or refinement of the meaning, I think SGD is happy with this annotation as-is.  The protein annotated isn't part of the RNAP core, but is required for nuclear localization of RNAPII.  I'll try to add the complex binding if I can find a paper that supports it, but from the paper that's up for this ticket there's not enough for me to say it's directly binding the RNAPII.

**Comment by pgaudet** [comment]
@suzialeksander thanks for looking into it ! 

Pascale

**Comment by ValWood** [comment]
Hi @suzialeksander , this would still be an incorrect use of colocalizes_with under the original guidelines. It was never intended to be used like this.

It will be culled anyway under this ticket
https://github.com/geneontology/go-annotation/issues/1940

If there is no evidence that the complex is bound, there should be no annotation.
(but the existing evidence used here is IPI?)

if it's role is binding and nuclear transport maybe you could use
https://www.ebi.ac.uk/QuickGO/term/GO:0140142
but is this clear from later papers. It seems that the role is in assembly (which might affect transport indirectly).

**Comment by ValWood** [comment]
https://www.ncbi.nlm.nih.gov/pubmed/26711263  for complex binding

**Comment by pgarmiri** [comment]
The  UniProt EBI annotations have been reviewed. The SIB curator has been noticed to look and remove the annotations (3 in total). The person is on holidays, so it might take some time for the changes to be made.

**Comment by pgaudet** [comment]
@suzialeksander 

I looked at the protein SGD has annotated to this term. I agree with @ValWood that this annotation is incorrect. 

I think A co_localizes  with B means that A goes to where B is. In this case you are looking at a nuclear import factor, so it's not localizing to the normal DNA-directed RNA polymerase II, core complex. 

In this case the term 'RNA polymerase II core binding" (GO:0000993)' would be appropriate. 

Thanks, Pascale

**Comment by suzialeksander** [comment]
Ah.  I missed that boat, thanks for that link.  I've deleted this annotation; I can't replace it because SGD doesn't want extraneous "protein binding" annotations and Protein2GO forces the reciprocal.  Thanks for the suggestions anyway.

TLDR: @pgaudet  I've removed it.

**Comment by pgarmiri** [comment]
Hi @suzialeksander, Protein2GO requires the reciprocal annotation only for annotations with IPI evidence code. I haven't looked into the paper but from what I read here, I think you can use IDA with the suggested annotation 'RNA polymerase II core binding" (GO:0000993)' and you can keep the functional information.  
Penelope

**Comment by pgaudet** [comment]
@pfey03 Did you have a chance to look at the Dicty annotation ?

**Comment by pgaudet** [comment]
I dont see the Dicty annotation anymore.
